.prettierrc.json

```diff
@@ -0,0 +1,10 @@
+{
+    "singleQuote": true,
```

Comment:
nit;

ngl I like double quotes personally

Reply:
I'm still single so it'll be single quotes.

Reply:
single quotes vote for me